.github/workflows/R-CMD-check.yaml

```diff
@@ -72,10 +81,20 @@ jobs:
           make all
 
       - name: Link shinylive assets
+        id: r-linked-assets
         shell: Rscript {0}
         run: |
+          shinylive_local_version <- shinylive:::package_json_version("shinylive_assets")
+          shinylive::assets_remove(shinylive_local_version)
           shinylive::assets_install_copy("shinylive_assets")
```

Comment:
We could use the py-shinylive assets version here, but that would lead to confusing testing. Instead, a proper and reasonable solution would be to update py-shinylive to support `--assets-version` in CLI to use different versions when exporting.